compiler/plc_diagnostics/src/diagnostician.rs

```diff
@@ -30,21 +30,28 @@ impl Diagnostician {
         file_name.and_then(|it| self.filename_fileid_mapping.get(it).cloned())
     }
 
-    /// assess and reports the given diagnostics
-    pub fn handle(&self, diagnostics: Vec<Diagnostic>) {
+    /// Assess and reports the given diagnostics.
+    ///
+    /// Note that the `handle` argument should always be [`None`] except for unit tests, where we have
```

Comment:
```suggestion
    /// Note that the handle` argument should always be [`None`] except for unit tests, where we have
```

Reply:
You are not, we removed it in a pairing session.